src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,41 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final int FIGURE_COUNT = 5;
+    private static final int MAX_VALUE = 10;
+    private final Random random = new Random();
+
+    public Figure getRandomFigure() {
+        ColorSupplier colorSupplier = new ColorSupplier();
+        String color = colorSupplier.getRandomColor();
+
+        switch (random.nextInt(FIGURE_COUNT)) {
+            case 0:
+                double squareSide = random.nextDouble() * MAX_VALUE;
```

Comment:
create separate method for getting `random.nextDouble() * MAX_VALUE` and use it for each type of figure